packages/koa-shopify-auth/src/auth/redirection-page.ts

```diff
@@ -12,7 +12,7 @@ export default function redirectionScript({origin, redirectTo, apiKey}) {
           var Redirect = AppBridge.actions.Redirect;
           var app = createApp({
             apiKey: '${apiKey}',
-            shopOrigin: '${origin}',
+            shopOrigin: '${encodeURI(origin)}',
```

Comment:
No idea how to 🎩 this but I think we can unit test this since it just returns a string that gets interpolated in JS.